manifest.json

```diff
@@ -1,26 +1,46 @@
 {
-  "name": "@deskpro-apps/my-app",
-  "title": "My App",
+  "name": "@deskpro-apps/google-calendar",
+  "title": "Google Calendar",
   "description": "My Deskpro app",
-  "version": "0.0.1",
+  "version": "1.0.0",
```

Comment:
the version is automatically bumped, if this is the first time setting it up, set it to 0.0.0 and add a major version label

Reply:
done